### PR TITLE
Initial documentation for the journey flow

### PR DESCRIPTION
**Pull request by ajaxscape**

### CODE REVIEW

**Comment by ajaxscape** [comment]
Please note that I have also changed the suffix of the nunjucks views to .view.njk so that they are inline with the naming standard of the views in the page definition pages.

File: docs/technical/journey-flow.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+to the start of that modules map and flows through that map.  When a next property within that map is set
+to return, the flow returns to the previous map and continues.
+
+As I have included no [branching](#branching) in the above map, I would expect the paths (pages) to be traversed in the following order:
+```text
+- /
+- /applicant/name
+- /applicant/address/search
+- /applicant/address/select
+- /applicant/address/select
```
Comment (disconnect78) [inline docs/technical/journey-flow.md]: This looks like a typo and it should be `/applicant/address/entry`

Comment (StuAA78) [inline docs/technical/journey-flow.md]: This looks like a typo that should be `/applicant/address/entry`

**Review by rudenoise — APPROVED** [review]
Nice write-up, can totally follow along. I'll use it for my current story :+1:

File: docs/technical/journey-flow.md (hunk 1 above)
Comment (ajaxscape) [reply docs/technical/journey-flow.md]: Fixed the typo, will now merge

**Pull request merged**